### PR TITLE
Update tip documents

### PR DESCRIPTION
**Pull request by AndreaFilyo**
Accessibility updates: Uploaded new images, created new md files for tipping flows and in-app pairing, replacing the existing PDF files. Removed the existing PDF files. Updated the Tipping support doc with links to the new md files.

### CODE REVIEW

File: Documentation/In-app Pairing Guide.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# In-app pairing guide
+## Background
+With the Zettle Reader 2 we are introducing a new type of Bluetooth called Bluetooth Low Energey (BLE). This enables us to do in-app pairing. This means that everything related to the Zettle reader is done in the application. The Zettle Reader 2 will not be visible in Bluetooth settings. 
+1. Go to **Zettle Settings** and select **Card readers** to connect the Zettle Reader 2 from the SDK.<br>
```
Comment (bobzettle) [inline Documentation/In-app Pairing Guide.md:4]: On Android we don't have a **Zettle Settings** view. You open the card readers settings directly.

File: Documentation/In-app Pairing Guide.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+<br>![In-app pairing step 1](images/In-app_1.png)
+2. The message “No card reader connected” displays.
+3. Tap **Connect card reader**.<br>
+<br>![In-app pairing step 2-3](images/In-app_2.png)
+4. Select **Zettle Reader 2**. Note that other card readers can be connected from the iOS Bluetooth settings.
```
Comment (bobzettle) [inline Documentation/In-app Pairing Guide.md:9]:
```suggestion
4. Select **Zettle Reader 2**. Note that other card readers can be connected from the Android Bluetooth settings.
```

File: Documentation/In-app Pairing Guide.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+5. The user is asked to turn on the Zettle Reader 2 for it to be detected. Detected readers appear in the list.
+6. The number displayed in the app should correlate to the number in the card reader display.<br>
+<br>![In-app pairing step 4-6](images/In-app_3.png)
+7. When a card reader is selected, the BLE pairing is initialized. 
+8. When a connection is established, the user will be asked to compare numbers in the app and in the card reader display.
+9. The user must confirm the numbers on both devices. This is done by selecting **Confirm** in the app, and clicking the confirmation button in the card reader.<br>
+<br>![In-app pairing step 7-9](images/In-app_4.png)
+10. When the pairing is confirmed, the reader displays in the **Connected card readers** view.
+11. To forget a card reader, go to the Card reader detail view. Note that forgetting a card reader can no longer be done from the iOS Bluetooth settings.<br>
```
Comment (bobzettle) [inline Documentation/In-app Pairing Guide.md:18]:
```suggestion
11. To forget a card reader, go to the Card reader detail view. Note that forgetting a card reader can no longer be done from the Android Bluetooth settings.<br>
```

File: Documentation/Tipping_flow_on_the_Zettle_card_reader/Total Amount Tipping Flow.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+3. The card reader prompts the customer to enter the payable amount including tip.
+4. The customer enters the total amount and clicks the confirmation button.<br>
+<br>![Total amount tipping flow steps 1-4](/Documentation/images/Total_Amount_2.png) 
+5. The total amount to be payed and “Tap/Insert card” displays in the POS user interface.
+6. “Tap/Insert card” displays on the card reader.
+7. The customer taps the card on the card reader.
+8. The “Approved” message displays on the card reader, followed by “Thank you”.<br>
+<br>![Total amount tipping flow steps 5-8](/Documentation/images/Total_Amount_3.png)<br>
+## Tipping amount is too high
+There is a maximum level validation to prevent accidental or invalid entering of amounts. This can happen for example if customers enter their PIN code instead of a tip amount. The maxium tip level is 35 %. If this is exceeded, the card reader will display a “Too high amount” warning message. The message will time out after three seconds.<br>
```
Comment (bobzettle) [inline Documentation/Tipping_flow_on_the_Zettle_card_reader/Total Amount Tipping Flow.md:23]: I don't think the max tip limit is an exact number? 
Isn't it different in different markets?

**Review by bobzettle — CHANGES_REQUESTED** [review]

File: Documentation/Tipping_flow_on_the_Zettle_card_reader/Total Amount Tipping Flow.md (hunk 4 above)
Comment (AndreaFilyo) [reply Documentation/Tipping_flow_on_the_Zettle_card_reader/Total Amount Tipping Flow.md:23]: OK, should I remove this part then? "The maximum tip level is 35 %"? (there was a typo there as well...)

Comment (AndreaFilyo) [reply Documentation/Tipping_flow_on_the_Zettle_card_reader/Total Amount Tipping Flow.md]: Removed mentioning of tipping level value, as discussed.

**Comment by AndreaFilyo** [comment]
Final fixes. Related Jira ticket: DEVX-641

**Pull request merged**